packages/react/src/components/Modal/Modal.stories.tsx

```diff
@@ -15,115 +17,128 @@ const decorators = [
 
 export default {
   title: 'Komponenter/Modal',
-  component: Modal.Dialog,
+  component: Modal,
   decorators,
 } as Meta;
 
-export const Preview: StoryFn<typeof Modal.Dialog> = (args) => {
-  return (
-    <>
-      <Modal.Root>
-        <Modal.Trigger>Open Modal</Modal.Trigger>
-        <Modal.Dialog {...args}>
-          <Modal.Header>Modal header</Modal.Header>
-          <Modal.Content>
-            <Paragraph>
-              Lorem ipsum dolor sit, amet consectetur adipisicing elit.
-              Blanditiis doloremque obcaecati assumenda odio ducimus sunt et.
-            </Paragraph>
-          </Modal.Content>
-          <Modal.Footer>Modal footer</Modal.Footer>
-        </Modal.Dialog>
-      </Modal.Root>
-    </>
-  );
-};
+export const Preview: StoryFn<typeof Modal> = (args) => (
+  <Modal.Context>
+    <Modal.Trigger>Open Modal</Modal.Trigger>
+    <Modal {...args}>
+      <Heading size='xs' spacing>
+        Modal header
+      </Heading>
+      <Paragraph spacing>
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Blanditiis
+        doloremque obcaecati assumenda odio ducimus sunt et.
+      </Paragraph>
+      <Paragraph size='sm'>Modal footer</Paragraph>
+    </Modal>
+  </Modal.Context>
+);
 
-export const WithoutTriggerComponent: StoryFn<typeof Modal.Dialog> = (args) => {
+export const WithoutModalContext: StoryFn<typeof Modal> = (args) => {
   const modalRef = useRef<HTMLDialogElement>(null);
 
   return (
     <>
       <Button onClick={() => modalRef.current?.showModal()}>Open Modal</Button>
-      <Modal.Root>
-        <Modal.Dialog {...args} ref={modalRef}>
-          <Modal.Header subtitle='Modal subtittel'>Modal header</Modal.Header>
-          <Modal.Content>
-            <Paragraph>
-              Lorem ipsum dolor sit, amet consectetur adipisicing elit.
-              Blanditiis doloremque obcaecati assumenda odio ducimus sunt et.
-            </Paragraph>
-          </Modal.Content>
-          <Modal.Footer>Modal footer</Modal.Footer>
-        </Modal.Dialog>
-      </Modal.Root>
+      <Modal.Context>
+        <Modal {...args} ref={modalRef}>
+          <Paragraph size='sm'>Modal subtittel</Paragraph>
+          <Heading size='xs' spacing>
+            Modal header
+          </Heading>
+          <Paragraph spacing>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Blanditiis
+            doloremque obcaecati assumenda odio ducimus sunt et.
+          </Paragraph>
+          Modal footer
+        </Modal>
+      </Modal.Context>
```

Comment:
```suggestion
      <Modal {...args} ref={modalRef}>
        <Paragraph size='sm'>Modal subtittel</Paragraph>
        <Heading size='xs' spacing>
          Modal header
        </Heading>
        <Paragraph spacing>
          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Blanditiis
          doloremque obcaecati assumenda odio ducimus sunt et.
        </Paragraph>
        Modal footer
      </Modal>
```
Let's make the story called `WithoutModalContext` not use a context